### ISSUE TITLE
Cant quite get it to work

### ISSUE THREAD

**Issue by bigmantingzz** [Siberpone/lazy-pony-prompter#13]
Hey im pretty stupid, trying to get LPP to work on comfyui. I half got it to work but im running into an issue. When i use the e621 node with "anthro female -comic -text solo" for example i have no issues. images get generated. Now when i try it with the danbooru node im running into issues. Running danbooru with "1girl" generates an image without issues. If i add anything else i get an error. For example if i add "1girl solo" i get  this : LPP_Danbooru
422 Client Error: Unprocessable Entity for url: https://danbooru.donmai.us/posts.json?tags=1girl+solo+rating%3Ae+order%3Arank&limit=200&page=1 

This is for sure me being dumb but help would be much appreciated 😭

**Comment by Siberpone** [comment]
tl;dr: danbooru sucks.

Apparently, they only allow a maximum of 2 tags in the search query, which is ridiculous. Unfortunately, there's nothing I can do about that. Note, that if "sort type" is set to anything other than "None" it automatically adds additional tag which counts towards the limit. "rating" doesn't seem to be counted, though.

**Comment by bigmantingzz** [comment]
yea that is super cringe, thanks for the quick reply tho and this is still a banger hope u add other sites like r34 etc in the future
